index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Birdwatching</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto+Condensed:300%7CCinzel+Decorative:700"
+      rel="stylesheet"
+    />
+    <link href="style.css" rel="stylesheet" type="text/css" />
+    <!--[if lt IE 9]>
+      <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+    <![endif]-->
+  </head>
+
+  <body>
+    <header>
```

Comment:
The `header` element should be outside the `body` element :) The structure should ideally be:

```suggestion
<header>header contents</header>
<body>body contents</body>
<footer>footer contents</footer>
```

This is important for users who rely on assistive technology to browse our sites. They need a proper layout structure for them to understand the page